.github/workflows/docker/sensei/install_sensei.sh

```diff
@@ -11,19 +14,21 @@ export CMAKE_GENERATOR=Ninja
 cd ${DOWNLOAD_ROOT}
 mkdir sensei
 cd sensei
-git clone https://gitlab.kitware.com/sensei/sensei.git &&
-cd sensei &&
-git checkout ${SENSEI_VERSION} &&
-mkdir build &&
-cd build &&
+git clone https://github.com/SENSEI-insitu/sensei.git
+cd sensei
+git checkout develop
```

Comment:
@kwryankrattiger We want this to be pinned to an official release (as it was before) so that the usual churn that occurs during sensei development doesn't break amrex builds.

Reply:
That wasn't possible at the time because the tagged release had a couple bugs in it that had to be fixed for this container to be built and work correctly in CI. The container that is used in CI is static, ie. this is not run each time CI is run, only when the container is rebuilt. The develop that was used to generate the container for amrex CI is the develop branch on 2022-06-13 which is denoted by the container name I pushed and referenced in the other PR for SENSEI updates. I agree it would be more ideal to use a tagged version, if you wanted to tag a new patch version of sensei, v4.0.1, I would be happy to update this and regenerate an image using the tagged version, I think most of the build/install issues have been fleshed out on develop now so that may be worthwhile anyway.

Reply:
OK. the container is static so this is not going to run against future changes. Sorry, I misunderstood the situation.